axelor-contract/src/main/java/com/axelor/apps/contract/service/ContractLineServiceImpl.java

```
@@ -171,6 +176,15 @@ public ContractLine computeTotal(ContractLine contractLine) {
     if (contractLine.getTaxLine() != null) {
       taxRate = contractLine.getTaxLine().getValue().divide(new BigDecimal(100));
     }
+
+    if (contractLine.getContractVersion() != null) {
+      try {
+        contractLine = computePricesPerYear(contractLine, contractLine.getContractVersion());
+      } catch (Exception e) {
+        TraceBackService.trace(e);
+      }
```

why do we need to silence all exceptions here ?